bigip/resource_bigip_ltm_monitor.go

```diff
@@ -148,10 +170,12 @@ func resourceBigipLtmMonitorRead(d *schema.ResourceData, meta interface{}) error
 			d.Set("manual_resume", m.ManualResume)
 			d.Set("parent", m.ParentMonitor)
 			d.Set("name", name)
+			d.Set("username", m.Username)
+			d.Set("password", d.Get("password").(string))
```

Comment:
Does the test pass then? It should fail if there would be changes detected after an apply.